subt/ros/robot/src/map_tf_broadcaster.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python2
+"""
+  Create ROS frame_id "map", which is global world position in DARPA coordinate system
+  (shifted and rotated so that (0, 0, 0) is in the middle of the entrance)
+"""
+from __future__ import print_function
+import time
+import sys
+
+import rospy
+#import tf
+import tf2_ros
```

Comment:
Interesting. What is the difference between this and the former one?

Reply:
https://answers.ros.org/question/11242/what-is-the-difference-between-tf-and-tf2/

_tf2 is a new implementation of tf with better abstraction and dependency isolation. See the wiki page for an overview: tf2 The design of the new version is documented on the tf2/Design page._

_tf2 is still experimental and in the geometry_experimental stack and it is still recommended to use the tf package until tf2 is released in the geometry stack._

This is note from 2018 for maybe it is already OK? As I was copy pasting ... so not sure, I can try older version?

Reply:
not sure if `tf` supports static transform broadcaster - used `tf2_ros` from some example